### DISCUSSION TITLE
Mirror terminal

### DISCUSSION THREAD

**Discussion by jaitra1264** [cockpit-project/cockpit#16760] [Q&A] [answered]
Is there a feature that allows mirroring the cockpit terminal to the host terminal?

**Comment by martinpitt** [comment, marked as answer]
What do you mean with "the" host terminal? Cockpit's terminal is "a" host terminal in the same way SSH or graphical sessions are -- PTYs. I suppose you mean the non-pseudo-terminals, such as VT1. But Linux does not have a concept of "mirroring" such a terminal, they are (deliberately) isolated from one another. If you need such functionality, please use screen, tmux, byoby or similar terminal sharing solutions.
